requirements/requirements.txt

```diff
@@ -39,7 +39,7 @@ pytz==2014.10
 PyYAML==3.10
 raven==3.1.17
 requests==2.0.0
-south==0.7.3
+south==1.0.2
```

Comment:
It's a migration in the tastypie app. It's added to INSTALLED_APPS here: https://github.com/dimagi/commcare-hq/pull/6794/files#diff-bdf3ecebd8379ca98cc89e545fc90899R193